README.md

```diff
@@ -4,13 +4,18 @@ Simple module to generate interactive console menus (like yeoman)
 # Examples:
 
 ```powershell
-menu @("option 1", "option 2", "option 3")
+$Options  = @("Option 1", "Option 2", "Option 3")
```

Comment:
I prefer "one liner" style, especially with animated GIF

Reply:
I have fixed the first two example gifs as well as the text example. 

The reason was that in Example 3 I use 3 options as well as a custom message which means the line gets too long as per the [style guidelines](https://docs.microsoft.com/en-us/powershell/scripting/community/contributing/powershell-style-guide?view=powershell-7.2#limit-line-length-to-100-characters) for PowerShell.